### ISSUE TITLE
[REQUEST] Make Quick Menu its own menu. - And some other small, related features.

### ISSUE THREAD

**Issue by devanteweary** [libretro/RetroArch#16909]
Hey there.

This is a feature request to move the Quick Menu out of the Main Menu and its own entry.  
Been using RA for years and to this day it still bugs my brain out slightly when I have to navigate between the settings menu and the quick menu.

I'm suggesting, for instance using the Ozone menu driver:

[Main Menu]
[Quick Menu] <-- also suggesting rename this to something like "Running Content" or "Currently Playing"]
[Settings]
[Exit] <-- Also suggesting add exit to the OUTSIDE of the Main Menu.

And lastly, the option to keep the sidebar on the screen while in the quick menu. At least in Ozone.

So in summary:
* Make Quick Menu its own thing.
* Rename Quick Menu to Currently Running or something like that.
* Add Exit as a last option.
* Option to keep Sidebar open while in the Quick Menu.

Thanks for reading!!

And just to lore it out a little, I just think it looks a little less confusing and is a little more intuitive.

**Comment by odditude42** [comment]
seconded.

promoting the quickmenu to the top level of the hierarchy could also help prevent new users from getting lost when trying to return to content.

i also agree that it should be renamed, but don't have a suggestion there. here's some other terms to maybe help spark a better idea:

* active content
* current content
* running content
* now playing

**Comment by hizzlekizzle** [comment]
I don't think renaming it is helpful. After all, Sony themselves calls their similar menu "Quick menu"

**Comment by devanteweary** [comment]
> I don't think renaming it is helpful. After all, Sony themselves calls their similar menu "Quick menu"

Just for the sake of conversation, Quick Menu just never really made sense. (my opinion only!)

When you think of titles, what is it conveying? Quick as opposed to what? The options in there aren't really available anywhere else so it's not a "quick" menu, it's the _only_ menu for those options.

Also, in a way it doesn't make sense because I get that it's the first menu you go into when you press the menu button, but looking at it from the root level, it's actually not "quick" as its under Main Menu so you have to click twice to get to the options within it.

But anyway, just talking to talk here. :P'

The main thing is moving it to outside of the main menu (and the Exit option as well! (not the close game content though).

Thank you for coming to my Ted talk!! :P'
